docs/pool.md

```diff
@@ -28,3 +28,25 @@ pool.disconnect()
 ```
 
 For more information about Peer events please read the [Peer](peer.md) documentation. Peer events are relayed to the pool, a peer event `inv` in the pool would be `peerinv`. When a peer is disconnected the pool will try to connect to the list of known addresses to maintain connection.
+
+## Trusted Peers
+
+By default, peers will be added via DNS discovery and as peers are announced in the network. To connect only to specific trusted peers, configurations options can be included:
+
+```javascript
+
+var pool = new Pool(Networks.livenet, {
+  discover: false,
+  listenAddr: false,
+  addrs: [
+    {
+      ip: {
+        v4: '127.0.0.1'
+      }
+    }
+  ]
+});
```

Comment:
Could you add comments to each option to make it clearer, please? (we want docs to be as obvious as possible)
Example:
```
var pool = new Pool(Networks.livenet, {
  discover: false, // prevent Pool from discovering and connecting to new peers
  listenAddr: false, // handle network addr messages 
  addrs: [ // initial peers to connect to
    {
      ip: {
        v4: '127.0.0.1'
      }
    }
  ]
});
```

btw: why do we have both `discover` and `listenAddr`? can we simplify that into 1 config setting?